### DISCUSSION TITLE
Why does it say `Abort` here?

### DISCUSSION THREAD

**Discussion by spamegg1** [VirtusLab/scala-cli#2605] [General]
Is it a `curl` thing?

```bash
$ scala-cli update                                                                                                  

Do you want to update scala-cli to version 1.1.0? [Y/n]

Abort # <- Right here. Every time!
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32.1M  100 32.1M    0     0  91875      0  0:06:06  0:06:06 --:--:--  100k
Successfully installed scala-cli 1.1.0
```

**Comment by Gedochao** [comment]
https://github.com/VirtusLab/scala-cli/blob/9f9562062cc9a2a890e1afdc18a4ed802f9d6251/modules/options/src/main/scala/scala/build/interactive/Interactive.scala#L55

You are running in interactive mode and I can't see the `y` in the log, so my best guess would be it's coming from here...
But then, the update then proceeds from there. So it is weird that it would be printed 🤔

**Reply by spamegg1** [reply]
Thanks. Yes it's weird and I'm concerned every time that something went wrong! I guess it's not a bug but maybe better reporting would be nice.
